cmd/nomos/status/client.go

```diff
@@ -354,6 +361,34 @@ func (c *ClusterClient) namespaceRepoClusterStatus(ctx context.Context, cs *Clus
 	}
 }
 
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and RootSync CRD
+// If RootSync CRD exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func (c *ClusterClient) IsOssInstallation(ctx context.Context, cs *ClusterState) bool {
+	v, cmErr := c.ConfigManagement.Version(ctx)
+	_, operatorDepErr := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, ACMOperatorDeployment, metav1.GetOptions{})
+
+	if cmErr != nil || (operatorDepErr != nil && !apierrors.IsNotFound(operatorDepErr)) {
+		cs.Error = "Failed to check for operator installation status"
```

Comment:
Suggest including the error in `cs.Error` to provide more details.

Reply:
Updated error message to include more details